api/chunks/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from pgvector.django import VectorField
+from typing import Tuple
+
+class Chunk(models.Model):
+    text: Tuple[str, str] = models.CharField(max_length=100)
+    embedding: Tuple[list[float], VectorField] = VectorField(dimensions=10)
+    chunk_idx: Tuple[int, int] = models.IntegerField()
+    start_char: Tuple[int, int] = models.IntegerField()
+    end_char: Tuple[int, int] = models.IntegerField()
```

Comment:
**issue:** This is not correct, see my previous comment. It may help you to set pyright to strict

Reply:
but your suggestion, didn't work for models. The only way i can see is somethinkg like this `text: str = models.CharField(max_length=100)`